src/ResourceManager/Automation/Commands.Automation/packages.config

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.Azure.Management.Automation" version="3.4.0-preview" targetFramework="net452" />
-  <package id="Microsoft.Azure.Management.ResourceManager" version="2.0.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.ResourceManager" version="1.9.1-preview" targetFramework="net452" />
```

Comment:
Shouldn't we just remove this altogether?

Reply:
Yes, I'm less concerned about doing that since these files (all .config files) will be removed in 2 weeks anyway. I just flat changed all Management.ResourceManager references to the same version (to be consistent).